tasks/post-to-cmr/index.js

```diff
@@ -101,10 +101,12 @@ function checkForMetadata(granules, cmrFiles) {
  * @param {boolean} event.config.skipMetaCheck - option to skip Meta file check
  * @param {Object} event.input.granules - Object of all granules where granuleID
  *    is the key
+ * @param {string} event.input.cmrRevisionId - CMR Revision ID
  * @returns {Promise<Object>} the promise of an updated event object
  */
 async function postToCMR(event) {
   // get cmr files and metadata
+  const revisionId = event.input.cmrRevisionId;
```

Comment:
I think we need to update the `post-to-cmr` input task schema to note that this an optional input field now?